### ISSUE TITLE
IMUNet: Efficient Regression Architecture for IMU Navigation and Positioning. (arXiv:2208.00068v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#21611]
IMUNet: Efficient Regression Architecture for IMU Navigation and Positioning. (arXiv:2208.00068v1 [cs.RO])<br>
https://ift.tt/aoZYEH7<br>
<p>Data-driven based method for navigation and positioning has absorbed attention in recent years and it outperforms all its competitor methods in terms of accuracy and efficiency. This paper introduces a new architecture called IMUNet which is accurate and efficient for position estimation on edge device implementation receiving a sequence of raw IMU measurements. The architecture has been compared with one dimension version of the state-of-the-art CNN networks that have been introduced recently for edge device implementation in terms of accuracy and efficiency. Moreover, a new method for collecting a dataset using IMU sensors on cell phones and Google ARCore API has been proposed and a publicly available dataset has been recorded. A comprehensive evaluation using four different datasets as well as the proposed dataset and real device implementation has been done to prove the performance of the architecture. All the code in both Pytorch and Tensorflow framework as well as the Android application code have been shared to improve further research.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
